google/cloud/testing_util/assert_ok.cc

```diff
@@ -24,9 +24,7 @@ testing::AssertionResult IsOkPredFormat(char const* expr,
     return testing::AssertionSuccess();
   }
   return testing::AssertionFailure()
-         << "Status of \"" << expr
-         << "\" is expected to be OK, but evaluates to \"" << status.message()
-         << "\" (code " << status.code() << ")";
+         << "Value of: " << expr << "\nExpected: is OK\nActual: " << status;
```

Comment:
s/Actual:/  Actual:/?

Reply:
Crap, I'm pressing buttons too fast. I didn't notice your comment, sorry.

Why the extra space? If you want me to indent it, then there should be one before "Expected:" too, no?

Reply:
Why the extra spaces (aligning the colons)?  Only because that is how the (pre) Abseil EXPECT_OK() does it.  Nice to have, but no biggie either way.